lib/datadog/tracing/contrib/action_pack/action_dispatch/instrumentation.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require_relative '../../../metadata/ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module ActionPack
+        module ActionDispatch
+          # Instrumentation for ActionDispatch components
+          module Instrumentation
+            module_function
+
+            def set_http_route_tag(http_route)
+              return if http_route.nil?
```

Comment:
Most callers of `set_http_route_tag` seem to call it with a string, so I'm not sure `http_route` will ever be nil. Should this check perhaps be for an empty string, rather than nil?

Reply:
yes, I think it will be better

Reply:
I removed this method for setting the route and replaced it with a simple method for route formatting here https://github.com/DataDog/dd-trace-rb/pull/3849/commits/137d110e3e90313c4d67554905d6a48893027e9f